src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -109,7 +106,7 @@ const BaseSubBudgetCard = ({
       to={`/${enterpriseSlug}/admin/${ROUTE_NAMES.learnerCredit}/${budgetId}`}
       variant={[BUDGET_STATUSES.expired, BUDGET_STATUSES.retired].includes(budgetLabel.status) ? 'outline-primary' : 'primary'}
     >
-      {isRetired ? (
+      {(status === BUDGET_STATUSES.retired || status === BUDGET_STATUSES.expired) ? (
```

Comment:
nit: similar to other comments, perhaps `[BUDGET_STATUSES.expired, BUDGET_STATUSES.retired].includes(status)`?

Actually, this line might be abstracted out to a `isRetiredOrExpired` type variable since there's multiple code paths relying on it? Might help DRY it up a bit.